### PR TITLE
Harry 0.4.0: New release

### PR DESCRIPTION
**Pull request by rieck**
The new release supports measuring string similarity at the granularity of bytes, bits and tokens. A Python interface has been added. Several minor bugs have been fixed.

### CODE REVIEW

**Comment by rieck** [comment]
Anything missing for this request to be merged and closed? Let me know.

File: harry.rb [hunk 1]
```diff
@@ -2,12 +2,13 @@
...
   homepage "http://www.mlsec.org/harry"
-  url "http://www.mlsec.org/harry/files/harry-0.3.2.tar.gz"
-  sha1 "8a4bbe03f278f3f9f55357f32728e738f95c7732"
+  url "http://www.mlsec.org/harry/files/harry-0.4.0.tar.gz"
+  sha256 "cb3526efbf119cae1de0c65745a40788fb1c483f95dab568ce75e5e222abff78"
 
   depends_on "pkg-config" => :build
   depends_on "libconfig"
   depends_on "libarchive" => :recommended
+  depends_on "homebrew/python/numpy" => :optional
```
Comment (iml) [inline harry.rb]: I think we're trying to get away from depending on brewed numpy (see https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Python-for-Formula-Authors.md). It's also not clear how the python interface is being enabled. Does it build with system Python and NumPy?

**Comment by rieck** [comment]
The Python interface runs with system and homebrew Python. It is enabled automatically, if Python is available. I removed the `numpy` dependency.

**Comment by iml** [comment]
@BrewTestBot test this please.

**Pull request closed without merging**